pyproject.toml

```diff
@@ -102,9 +102,9 @@ lint.select = ["E", "F", "W"]
 
 
 [tool.pytest.ini_options]
- addopts = '-s -vvv --cache-clear --cov-report=term-missing --cov --cov-fail-under=75'
- markers = [
-     "smoke: quick tests to check basic functionality",
-     "sanity: detailed tests to ensure major functions work correctly",
-     "regression: tests to ensure that new changes do not break existing functionality"
- ]
+addopts = '-s -vvv --cache-clear --cov-report=term-missing --cov --cov-fail-under=75'
+markers = [
+    "smoke: quick tests to check basic functionality",
+    "sanity: detailed tests to ensure major functions work correctly",
+    "regression: tests to ensure that new changes do not break existing functionality"
+]
```

Comment:
My bad leaving this in! Thanks for removing, I was just trying to make working on tests easier